src/styles/variables.scss

```diff
@@ -0,0 +1,23 @@
+@mixin cell-propertis ($cell-width, $cell-height, $bg-color) {
```

Comment:
Mixins are not variables, so you should keep them in a separate file. 
There is also a typo in the name now (should be `cell-properties`).